### ISSUE TITLE
Grindstone Disenchanting: Book splitting

### ISSUE THREAD

**Issue by Cynosphere** [exaskye/Fabrication-Features#84]
Enchanted book with multiple enchants + book = first enchantment moved to new book. Most likely should be listed as an extra just to keep some slight balance.

**Comment by exaskye** [comment]
Yeah, currently just about the only tradeoff to Grindstone Disenchanting is that it moves all enchantments.

On my server we've been using a Broken Gear Drops Components config that always dumps enchantments to perform splitting.
